back-end/dist/DB/memoryDB.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
```

Comment:
Why would you commit `dist` folder?

Reply:
For the sake of easy deployment, l have some modification in index.html, but this increment was not taken into account due to .gitignore. l know it's not a good practice, but it's a fast and easy way.

There are many others solutions like do more config in webpack (permanently)
 or temporally transfer the files by ssh etc...

Reply:
What about build it into a docker image?

Reply:
Yes, l does another good way. Push it to a public docker hub repo and pull,run it directly on the host server.